docs/gravity-bridge/gravity-bridge-relayer-modes.md

```diff
@@ -0,0 +1,13 @@
+## Gravity Bridge Relayer Modes
+
+Gravity Bridge relayer supports the following modes:
+1. AlwaysRelay
+2. Api
+3. File
+
+
+### Always relay mode
+
+### Api mode
+
```

Comment:
```suggestion

The env variable `RELAYER_API_URL` needs to be set to use this mode. The relayer will call an API that will estimate the cost of sending the batch, check if the daily limit is reached and none of the addresses are blacklisted. It will send the batch only if all the conditions are met.
```